source/includes/administration/_invoices.md

```diff
@@ -184,3 +184,90 @@ Attributes | &nbsp;
 `detail.categories.products.unit`<br/>*Object* | The unit object of the product.
 `detail.categories.products.unit.unit`<br/>*Object* | The unit value of the product.
 `detail.categories.products.unit.name`<br/>*Object* | The name of the unit of the product in the required language. Only present when defining custom units.
+
+<!-------------------- APPROVE INVOICE -------------------->
+### Approve invoice
+
+`PUT /invoices/:invoice_id/approve`
+
+Manually approve a draft invoice and issue the invoice to the customer by email.
```

Comment:
Do you think we should add what will happen if the invoice is not in `draft` state?

Reply:
https://github.com/cloudops/cloudmc-api-docs/pull/418/commits/f116010e0dd55c35d39fda4a440e0d90944dd099